cdap-api-common/src/main/java/co/cask/cdap/api/data/format/StructuredRecord.java

```diff
@@ -58,6 +58,15 @@ public Schema getSchema() {
   }
 
   /**
+   * Get the fields in the record.
+   *
+   * @return the fields in the record.
+   */
+  public Map<String, Object> getFields() {
```

Comment:
Do we need to add this method? Can't we get the schema and then get fields from it?